com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/StageAppStep.java

```diff
@@ -45,6 +63,24 @@ protected StepPhase executeAsyncStep(ExecutionWrapper execution) {
         }
     }
 
+    private StepPhase stageAppCF(ExecutionWrapper execution, CloudControllerClient client, CloudApplication app) {
```

Comment:
Why is the logic different for CF and XSA? Isn't that the whole purpose of the `StagingResolver`.